package.json

```diff
@@ -28,16 +28,21 @@
     "partial-lint": "git diff --name-only master... | grep js$ | xargs ./node_modules/.bin/eslint",
     "check": "npm run lint && npm run test",
     "tempcheck": "npm run partial-lint && npm run test",
-    "test": "jest"
+    "test": "jest",
+    "watch": "jest --watch"
```

Comment:
What is this used for?

Reply:
If you run npm run watch it'll watch your files for changes and every time you save a change it will retest your code to make sure you haven't broken anything. Extremely useful for development